src/components/base/image/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import propTypes from "prop-types";
+
+function Image({ src, alt, className }) {
+  return (
+    <div>
+      <img
+        src={require(`../../../assets/images/${src}`)}
```

Comment:
We can import the src on the top and use that inside require. Also, keep in mind about relative paths